dataclass_factory/parsers.py

```diff
@@ -345,6 +345,8 @@ def create_parser_impl(factory, schema: Schema, debug_path: bool, cls: Type) ->
         return get_literal_parser(factory, cls.__values__)
     if is_optional(cls):
         return get_optional_parser(factory.parser(cls.__args__[0]))
+    if cls is Pattern:
```

Comment:
lets's move to standard schemas like we did with datetime.

Reply:
Fixed